MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1283,7 +1283,7 @@ public string GetVideoQualityParam(EncodingJobInfo state, string videoEncoder, E
 
             if (string.Equals(videoEncoder, "h264_v4l2m2m", StringComparison.OrdinalIgnoreCase))
             {
-                param += " -pix_fmt nv21";
+                param += " -pix_fmt yuv420p";
```

Comment:
Can you remove this field and add a new case for V4L2 encoder after these lines?
https://github.com/jellyfin/jellyfin/blob/939d96e96ca6e3bc30a815e81bfc185eb29a32ad/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs#L2722-L2725

Reply:
typo? nv12 / nv21

Reply:
@nyanmisaka I submitted a new PR (#7291) that moves this change down to that location, however I did not fix the potential issue that @Maxplosion pointed out? All credit to @MELSunny as this is more or less just an update on his PR

Reply:
Thank you very much for your PR @aolszowka. 
nv12 is tested to work on raspberry pi 4 as well. 
@Maxplosion  nv21 is not a typo, it exists in pixel format in FFmpeg. But nv21 is not supported by raspberry pi encoder h264_v4l2m2m.

Reply:
> Can you remove this field and add a new case for V4L2 encoder after these lines?
> 
> https://github.com/jellyfin/jellyfin/blob/939d96e96ca6e3bc30a815e81bfc185eb29a32ad/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs#L2722-L2725

Many thanks for the contribution by @aolszowka, 
I have updated the code according to @nyanmisaka advice 

https://github.com/jellyfin/jellyfin/pull/7227/commits/deae9bb0933c65fa2e5eaccefc7b1a5e9a70afeb